fastlane/swift/SocketClient.swift

```diff
@@ -279,24 +309,35 @@ extension SocketClient: StreamDelegate {
         verbose(message: "response is: \(responseString)")
         switch socketResponse.responseType {
         case .clientInitiatedCancel:
-            self.socketDelegate?.commandExecuted(serverResponse: .clientInitiatedCancelAcknowledged)
-            self.closeSession(sendAbort: false)
+            self.socketDelegate?.commandExecuted(serverResponse: .clientInitiatedCancelAcknowledged) {
+                self.closeSession(sendAbort: false)
+                print("Leave \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
                verbose(message: "Leave \(#function) \(#line)")
```